api/v1alpha1/flowcollector_types.go

```diff
@@ -446,6 +446,11 @@ type FlowCollectorConsolePlugin struct {
 	// imagePullPolicy is the Kubernetes pull policy for the image defined above
 	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
 
+	//+kubebuilder:default:=false
+	// sendAuthToken is a flag to enable or disable forwarind auth token
+	// if enabled, this overide loki.sendAuthToken
+	ForwardUserAuthToken bool `json:"forwardUserAuthToken,omitempty"`
```

Comment:
Following the API recommendations, discouraging the use of bools, maybe I'd move this to a string property. Something like: `userAuthToken: {BLOCK | FORWARD}`

Reply:
maybe we should merge with `FlowCollectorLoki.SendAuthToken` to a single enum then ? :thinking:

Reply:
I change the value to an enum.

Reply:
> maybe we should merge with `FlowCollectorLoki.SendAuthToken` to a single enum then ? thinking

`FlowCollectorLoki.SendAuthToken` apply to both FLP and Console plugin while this new value only apply to the Console Plugin.

Also `FlowCollectorLoki.SendAuthToken` is still required for FLP when the user token forwarding is enabled.

Reply:
Yes. My thinking was about having `FlowCollectorLoki.SendAuthToken` as:
```yaml
enum:
  - IGNORE
  - SERVICE_ACCOUNTS
  - MIXED
  type: string
```

With proper documentation to explain `MIXED` behavior

Reply:
Done